wicker/core/config.py

```diff
@@ -29,22 +42,31 @@ def from_json(cls, data: Dict[str, Any]) -> WickerAwsS3Config:
 class WickerConfig:
     raw: Dict[str, Any]
     aws_s3_config: WickerAwsS3Config
+    wandb_config: WickerWandBConfig
 
     @classmethod
     def from_json(cls, data: Dict[str, Any]) -> WickerConfig:
         return cls(
             raw=data,
             aws_s3_config=WickerAwsS3Config.from_json(data["aws_s3_config"]),
+            wandb_config=WickerWandBConfig.from_json(data.get("wandb_config", {})),
         )
 
 
 _CONFIG: Optional[WickerConfig] = None
+_PREV_WICKER_CONFIG_PATH: Optional[str] = None
```

Comment:
We don't support it and it can't happen in operation unless the user writes their own code to inject a different env variable in during operation after instantiation has already occurred.
Here's a breakdown of the problem I'm trying to solve:
The config is loaded when the `datasets` module is loaded and set to the global and then it's loaded again by other modules but they just access the config already set instead of double loading. This has two problems I encountered doing this,
1. You can't set the config for individual tests easily at all and tests that are dependent on config sets, ie: testing config parsing or usage of different configs, cannot operate because that global never resets in the test environment. 
2. We have tests that couldn't access that config because it was using relative pathing, it was reliant on the specific module getting loaded first to not throw a file not found exception.

Reply:
I'll think about how to address this better and get back to you on it